vars/MPLModule.groovy

```diff
@@ -69,6 +70,11 @@ def call(String name = env.STAGE_NAME, Map cfg = null) {
     MPLManager.instance.pushActiveModule(module_path)
     Helper.runModule(module_src, module_path, [CFG: Helper.flatten(cfg)])
   }
+  catch( FlowInterruptedException ex ) {
+    //The exception is used by Jenkins to abort a running build and consequently
```

Comment:
Could you please also add a space between '//' and the actual comment - it's hard to read when they are too close.